akka-cluster-tools/src/main/scala/akka/cluster/singleton/ClusterSingletonManager.scala

```diff
@@ -472,6 +472,11 @@ class ClusterSingletonManagerIsStuck(message: String) extends AkkaException(mess
  *   Note that [[akka.actor.PoisonPill]] is a perfectly fine
  *   `terminationMessage` if you only need to stop the actor.
  *
+ *   '''Caution:''' care must be taken when stopping persistent actors as
+ *   [[akka.actor.PoisonPill]] can be processed faster than actor persists
+ *   its state, leading to possibly corrupt journal.
+ *   See also [[https://doc.akka.io/docs/akka/current/persistence.html#safely-shutting-down-persistent-actors]]
+ *
```

Comment:
Not sure this belongs here even if it is true.

Reply:
Should I remove it?